parsers/AR.py

```diff
@@ -131,10 +165,26 @@ def non_renewables_production_mix(zone_key: str, session: Session) -> Dict[str,
         }
         for production_info in sorted_production_list
     }
-
     return non_renewables_production
 
 
+def get_region_id(zone_key, session: Session) -> int:
+    """Fetches the region id for the zone that is required to get the production of that zone."""
+    regions_response = session.get(CAMMESA_REGIONS_ENDPOINT)
+
+    assert (
+        regions_response.status_code == 200
+    ), "Exception when fetching regions for AR: " "error when calling url={}".format(
+        CAMMESA_REGIONS_ENDPOINT
+    )
```

Comment:
I would prefer if this used `f"string {object}"` instead, it's a little easier to work with and it's sligltly faster from a preformance point of view.